user/service.go

```diff
@@ -0,0 +1,25 @@
+package user
+
+import "context"
+
+// Service is a type of service that manages business process
+type Service struct {
```

Comment:
If there is no business logic, should we skip Service struct and go straight to UserRepo? wdyt?
We can always introduce service layer when business logic arises.

Im still thinking if it is a good idea to create service just for the sake of having a service. 🤔

Reply:
agree with this